kotlin-guice/src/test/kotlin/com/authzee/kotlinguice4/KotlinBinderSpec.kt

```diff
@@ -140,7 +141,7 @@ object KotlinBinderSpec : Spek({
                 })
 
                 val iterable = injector.getInstance(key<Iterable<A>>())
-                iterable shouldEqual null
+                iterable?.shouldBeNull()
```

Comment:
Won't the null coalescing prevent it from running the actual assertion? What this test actually is trying to confirm is that `getInstance` doesn't throw a provisioning exception, so technically the assertion isn't really necessary I guess.

Reply:
Yeah, you're totally right. The test was failing because of shouldEqual doesn't accept null as this. My idea was to leave something in there do "document" what the test was trying to do. I'm happy to remove the assertion and the test will pass as long as there's no provision exception.